SetJointGainsMsg command
---
SetJointGainsMsg command
bool state
